modin/backends/pandas/query_compiler.py

```diff
@@ -835,6 +835,151 @@ def resample_var(self, resample_args, ddof, *args, **kwargs):
+            df.rolling(*rolling_args).std(ddof=ddof, *args, **kwargs)
+        )
+    )
+
+    rolling_rol_count = FoldFunction.register(
+        lambda df, rolling_args: pandas.DataFrame(df.rolling(*rolling_args).count())
+    )
+
+    rolling_rol_sum = FoldFunction.register(
+        lambda df, rolling_args, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).sum(*args, **kwargs)
+        )
+    )
+
+    rolling_rol_mean = FoldFunction.register(
+        lambda df, rolling_args, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).mean(*args, **kwargs)
+        )
+    )
+
+    rolling_rol_median = FoldFunction.register(
+        lambda df, rolling_args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).median(**kwargs)
+        )
+    )
+
+    rolling_rol_var = FoldFunction.register(
+        lambda df, rolling_args, ddof, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).var(ddof=ddof, *args, **kwargs)
+        )
+    )
+
+    rolling_rol_std = FoldFunction.register(
+        lambda df, rolling_args, ddof, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).std(ddof=ddof, *args, **kwargs)
+        )
+    )
+
+    rolling_rol_min = FoldFunction.register(
+        lambda df, rolling_args, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).min(*args, **kwargs)
+        )
+    )
+
+    rolling_rol_max = FoldFunction.register(
+        lambda df, rolling_args, *args, **kwargs: pandas.DataFrame(
+            df.rolling(*rolling_args).max(*args, **kwargs)
+        )
+    )
+
+    def rolling_rol_corr(self, rolling_args, other, pairwise, *args, **kwargs):
+        if len(self.columns) > 1:
+            return self.default_to_pandas(
+                lambda df: pandas.DataFrame.rolling(df, *rolling_args).corr(
+                    other=other, pairwise=pairwise, *args, **kwargs
+                )
+            )
+        else:
+            return FoldFunction.register(
+                lambda df: pandas.DataFrame(
+                    df.rolling(*rolling_args).corr(
+                        other=other, pairwise=pairwise, *args, **kwargs
+                    )
+                )
+            )(self)
+
+    def rolling_rol_cov(self, rolling_args, other, pairwise, ddof, **kwargs):
+        if len(self.columns) > 1:
+            return self.default_to_pandas(
+                lambda df: pandas.DataFrame.rolling(df, *rolling_args).cov(
+                    other=other, pairwise=pairwise, ddof=ddof, **kwargs
+                )
+            )
+        else:
+            return FoldFunction.register(
+                lambda df: pandas.DataFrame(
+                    df.rolling(*rolling_args).cov(
+                        other=other, pairwise=pairwise, ddof=ddof, **kwargs
+                    )
+                )
+            )(self)
+
+    rolling_rol_skew = FoldFunction.register(
```

Comment:
Can we put all of the `def` functions together and all of the `rolling_x = FoldFunction` functions together to conserve vertical whitespace?

Reply:
fixed